R/utils.R

```diff
@@ -410,8 +410,8 @@
 #' @noRd
 .parse_taxonomy <- function(
     taxa_tab, sep = "; |;", col.name = column_name, column_name = "Taxon",
-    remove.prefix = prefix.rm, prefix.rm = removeTaxaPrefixes, removeTaxaPrefixes = FALSE,
-    returned.ranks = TAXONOMY_RANKS, ...) {
```

Comment:
returned.ranks control which taxonomy ranks are returned in taxonomy table. I think it is best to return all ranks that have data.